### DISCUSSION TITLE
change the port for the admin account

### DISCUSSION THREAD

**Discussion by CamelT0E** [dani-garcia/vaultwarden#2957] [Ideas]
I know that you can deactivate the admin account by removing the admin token. wouldn't it be better to put the admin account on a different port? This would make the admin account only accessible for the internal network or only for certain ips. it would be more practical and the normal users would not be able to get to the admin page.

**Comment by BlackDex** [comment]
I suggest to use a reverse proxy for this. It can do all that and more like adding an extra basic auth on top of it for example, or filter IP's.

**Comment by CamelT0E** [comment]
Thanks for the answer

I use Vaultwarden on the Synology Diskstation with the built-in reverse proxy. There are no other settings there.

**Comment by dkebler** [comment]
I don't have a synology.  But reading a bit on the OS seems like it supports running docker containers.  You  could run an alternative webserver for RP like my favorite Caddy in a container on ports 80 443 (disabling the current one).  Or if you have another server running elsewhere put caddy on there and RP from it to containers/ports on your synology.     Caddy will support what @BlackDex suggests
